### DISCUSSION TITLE
Question: automaticLayout

### DISCUSSION THREAD

**Discussion by keeryuwa** [dineug/erd-editor#272] [Q&A]
Hello! Сan you please tell me how to use automatic layout when loading data using loadsqlddl()?
If I use editor.current.automaticLayout = true, then all tables are lined up and the links are visually confused.

**Comment by dineug** [comment]
The automaticLayout property is:
Width, height Alternate property.

Because there is no table autosort algorithm.
Very simply positioning the table.
